src/models/good-category.model.ts

```diff
@@ -15,45 +16,51 @@ export class GoodCategory {
   /**
    *
    */
-  @property({required: true, jsonSchema: {
-  type: 'string',
-}})
-  id: string;
+  @property({
+    id: true,
+    jsonSchema: {
+      type: 'string',
```

Comment:
Probably better to have it decorated as an integer

```suggestion
      type: 'number',
```